src/devices/Mcp960X/README.md

```diff
@@ -0,0 +1,13 @@
+﻿# Mcp960X - cold-junction compensated thermocouple to digital converter
```

Comment:
please use capital casing for the device name here, also add "family" (of devices) somewhere in the title so that `X` becomes more apparent what it means

Reply:
README.md updated